src/YieldDisburser.sol

```diff
@@ -467,4 +419,13 @@ contract YieldDisburser is OwnableUpgradeable {
 
         blockTime = _blockTime;
     }
+    /**
+     * @notice Set a new cycle length
+     * @param _cycleLength New cycle length
+     */
+
+    function setCycleLength(uint256 _cycleLength) public onlyOwner {
+        if (_cycleLength == 0) revert MustBeGreaterThanZero();
```

Comment:
line break after "checks" to be consistent with the rest of the contract

Reply:
I don't understand, this seems to be consistent with your test wrapper from what I can see (1 line break after "}") 🤔🧐

Reply:
addressed with 89bee13